friskby/views/quick.py

```diff
@@ -5,17 +5,37 @@
 from django.views.generic import View
 from django.utils import formats
 
+from dateutil import parser as dt_parser
+from datetime import datetime as dt
+
 from plot.models import *
 from sensor.models import *
 
 def make_timestamp( row ):
     row["timestamp_data"] = TimeStamp.create( row["timestamp_data"] )
     return row
 
+def sparsify(lst, minutes = 15, cutoff = 100):
+    if not lst:
+        return []
+    ret = []
+    prev = None
+    ret.append(lst[0])
+    prev = lst[0]['timestamp_data']
+    for e in lst:
+        delta = e['timestamp_data'] - prev
+        if delta.total_seconds() > minutes*60:
+            e['value'] = min(cutoff, e['value'])
```

Comment:
But I wanted `>100` to mean that the sensor is saturated and thus an actual point in the graph (however misleading) feels better than a hole in the graph.